src/main/java/org/openstreetmap/atlas/geography/PolyLine.java

```diff
@@ -740,7 +752,7 @@ public Angle maximumAngle()
 
     public Location middle()
     {
-        return offsetFromStart(Ratio.HALF);
+        return this.offsetFromStart(Ratio.HALF);
```

Comment:
Hi @matthieun , I don't use save actions at all. I think that's from @lucaspcram commit. Is it ok to keep this, or do you want it removed?

Reply:
Lol I knew this would come up. I screwed up a save action config at some point and those `this` keywords ended up everywhere without me noticing. The problem is that you can only remove them manually....

Reply:
Looks like there are about 30 places this happened. I don't mind fixing it myself (since it's my fault), but I am unsure how I can edit @hallahan 's PR. Ideas?

Reply:
@lucaspcram I added you as a collaborator to my fork, you can commit to the tippecanoe branch, which will update this pr.

https://github.com/hallahan/atlas/invitations

https://github.com/hallahan/atlas/tree/tippecanoe

Reply:
Mostly fixed. Looks like in some places of `PolyLine`, there were already `this` prefixes before this PR, since now the diff is showing some as removed (Please just ignore the commit noise, I don't want to spend time figuring out which `this` prefixes I added and which were already present before the PR). Also, there are a couple other spots in this PR where you used a `this` prefix where it was not necessary. I have left those alone. I think it's fine.